dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -96,9 +96,9 @@ dependent_startup_wait_for=rsyslogd:running
 {% if not skip_xcvrd %}
 [program:xcvrd]
 {% if delay_xcvrd %}
-command=bash -c "sleep 30 && {% if API_VERSION == 3 and 'xcvrd' not in python2_daemons %}python3 {% else %} python2 {% endif %}/usr/local/bin/xcvrd"
+command={% if skip_cmis_mgr %} bash -c "sleep 30 && python3 /usr/local/bin/xcvrd --skip_cmis_mgr" {% else %} bash -c "sleep 30 && python3 /usr/local/bin/xcvrd" {% endif % }
```

Comment:
the code refactor seems not having same criteria as previous one. can you explain why skip_cmis_mgr can replace python3 check?

Reply:
@lguohan platform-common and platform-daemon has moved to using python3. We no longer support python2

Reply:
why do we need to skip cmis_mgr for sn2700? we are not going to support cmis for sn2700 in the future?